pkg/apis/extensions/types.go

```diff
@@ -676,6 +676,12 @@ type PodSecurityPolicySpec struct {
 	// If set to false the container may run with a read only root file system if it wishes but it
 	// will not be forced to.
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem,omitempty"`
+	// SeccompProfiles lists the allowed profiles that may be set for the pod or
+	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles may
+	// be specifid by the pod or container.	The wildcard '*' may be used to allow all profiles.  When
+	// used to generate a value for a pod the first non-wildcard profile will be used as
```

Comment:
@liggitt that's what I mean. See my comment above. Existing PSPs have `nil` and that should allow the profiles that were permitted before. At the moment it is proposed that `nil` mean "no profile are allowed".

Pods without seccomp annotations are not problematic anyway regarding the PSP `nil` semantics.